e2e-tests/helpers/boundary-http/credentials.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { nanoid } from 'nanoid';
+import { checkResponse } from './responseHelper.js';
+import { readFile } from 'fs/promises';
+
+/**
+ * Creates a new ssh keypair credential
+ * @param {import('@playwright/test').APIRequestContext} request
+ * @param {string} credentialStoreId ID of the credential store where the credential will be created
+ * @param {string} username Username of the user credential
+ * @param {string} sshKeyPath Path to private key of the user credential
+ * @returns {Promise<Serializable>}
+ */
+export async function createStaticCredentialKeyPair(
+  request,
+  { credentialStoreId, username, sshKeyPath },
+) {
+  const privateKey = await readFile(sshKeyPath, {
+    encoding: 'utf-8',
+  });
+
+  const response = await request.post(`/v1/credentials`, {
```

Comment:
@ZedLi 
Perhaps outside the context of this PR, but request has been deprecated and its in maintenance mode, [here link to the info](https://github.com/request/request/issues/3142).
Did we consider using other package to perform http requests? I am not saying we shouldn't use request at all, isn't rocket science and it solves our need, but asking if we consider others due to the request situation.

Reply:
lol sorry for my fast wrong assumption!